cmd/cluster/support/common.go

```diff
@@ -3,9 +3,14 @@ package support
 import (
 	"errors"
 	"fmt"
+	"os"
+
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 	ctlutil "github.com/openshift/osdctl/pkg/utils"
-	"os"
+)
+
+var (
+	userParameterNames, userParameterValues []string
```

Comment:
I'd prefer if these variables were scoped to the specific function in which they're needed instead of being package scoped. Package scoped variables can easily bleed state between commands and make issues harder to debug.